README.md

```diff
@@ -0,0 +1,238 @@
+# python-aodndata
+This repository holds per-facility Python pipeline code, for example handlers, destination path functions and any other code specific to a given facility and/or pipeline.
+
+The code extends the [aodncore](https://github.com/aodn/python-aodncore) project which is treated as an upstream library, providing primarily the HandlerBase parent class, which provides all of the common handler capabilities and essentially implements the entire "generic handler".
+
+Facility/pipeline specific handlers are then created as a subclass of HandlerBase, and act to configure, extend and/or modify the behaviour as appropriate for the specific pipeline.
+
+For example, a child class may do one or more of the following:
+* define how the destination path is determined for the file(s) being handled (i.e. a dest_path function)
+* restrict which file extensions/types are allowed to be handled
+* determine which [compliance checks](https://github.com/ioos/compliance-checker) are performed against files (if applicable)
+* in case of a "multiple file" handler (e.g. a ZIP or manifest file), determine which files are included/excluded, e.g. process all NC files in a ZIP but not TXT files, or define on an individual file basis which ones are harvested and/or uploaded and/or archived
+* determine who is notified in case of handler error (or success)
+
+# User Guide
+
+It is highly recommended to use the [PyCharm IDE](https://www.jetbrains.com/pycharm/) for developing on this code base, as it provides many useful features out of the box, such as good unit test integration, real time code quality checking, automatic completion, and the usual basic IDE debugging capabilities such as setting breakpoints and syntax highlighting.
+
+## Setting up the development environment
+
+### Pre-requisites
+1. Ensure the [aodn/chef](https://github.com/aodn/chef) repository is checked out (directory will be referred to as **${CHEF_REPO}**), and the usual pre-requisites for running the PO box are met
+1. Ensure that your GitHub keys are in place so that you have write access to the **python-aodndata** repository on GitHub
+1. Provision the PO box using the **bin/po-box.sh** script, so that the required repositories are checked out and the PO box is ready for use
```

Comment:
Currently we still have to use the **po_pipeline_2** box (launched with `bin/po-pipeline-2-box.sh`). Merging this back into the original **po** box is TBD.

Reply:
Yep, I wrote it from the assumption that the merge had occurred.